extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -58,22 +57,11 @@ export class AmpScrollableCarousel extends BaseCarousel {
 
     this.container_ = this.element.ownerDocument.createElement('div');
     this.container_.classList.add('-amp-scrollable-carousel-container');
-    st.setStyles(this.container_, {
-      'white-space': 'nowrap',
-      'overflow-x': 'auto',
-      'overflow-y': 'hidden',
-      '-webkit-overflow-scrolling': 'touch',
-    });
     this.element.appendChild(this.container_);
 
     this.cells_.forEach(cell => {
       this.setAsOwner(cell);
       cell.classList.add('amp-carousel-slide');
-      st.setStyle(cell, 'display', 'inline-block');
-      if (cell != this.cells_[0]) {
-        // TODO(dvoytenko): this has to be customizable
-        st.setStyle(cell, 'marginLeft', '8px');
```

Comment:
The `marginLeft` for normal `.-amp-scrollable-carousel-container .amp-carousel-slide` is missing?